.github/renovate.json5

```diff
@@ -29,6 +29,16 @@
       "matchPackagePrefixes": ["com.linecorp.armeria:"],
       "groupName": "armeria packages"
     },
+    {
+      // prevent update to 2.4-groovy-4.0-SNAPSHOT
+      "matchPackageNames": ["org.spockframework:spock-bom"],
+      "allowedVersions": "!/\\-SNAPSHOT$/"
```

Comment:
I wonder how it even got a snapshot, we're not using any snapshot repos in our repositories config

Reply:
It somehow ends up searching in https://oss.sonatype.org/content/repositories/snapshots Perhaps there is a way to exclude that repository?